lib/active_interactor/rails/models.rb

```diff
@@ -27,19 +20,21 @@ def acts_as_context
         end
       end
 
+      # Model instance helper methods
+      # @author Aaron Allen <hello@aaronmallen.me>
+      # @since 1.0.0
       module InstanceMethods
-        # Override ActiveRecord's initialize method to ensure
+        # Override initialize method to ensure
         #  context flags are copied to the new instance
-        # @param context [Hash|nil] attributes to assign to the class
-        # @param options [Hash|nil] options for the class
-        def initialize(context = nil, options = {})
-          copy_flags!(context) if context
-          copy_called!(context) if context
-          attributes = context.to_h if context
-          super(attributes, options)
+        # @param attributes [Hash|nil] attributes to assign to the class
+        def initialize(attributes = nil)
+          copy_flags!(attributes) if attributes
+          copy_called!(attributes) if attributes
+          attributes_as_hash = attributes && attributes.respond_to?(:to_h) ? attributes.to_h : attributes.attributes&.to_h
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use safe navigation (`&.`) instead of checking if an object exists before calling the method.](https://app.codacy.com/manual/aaronmallen/activeinteractor/pullRequest?prid=4858256)